docs/ECO-METADATA.md

```diff
@@ -110,3 +110,182 @@ The frequency at which EVE updates location information is configurable using th
+(modem is online and connected), `radio-off` (modem has disabled radio transmission), `offline`
+(modem is offline), `unrecognized` (unrecognized operating mode).
+
+The set of SIM cards inserted into the modem is listed under `sim-cards`. Included is
+the identification number of the subscriber (`IMSI`) and the SIM card itself (`ICCID`).
+
+If EVE fails to configure modem and put it into a desired state, `config-error` will explain
+what exactly went wrong.
+If enabled, EVE will periodically test connectivity by running a ping towards a remote
+endpoint (with configurable address). If the last probing failed, `probe-error` will contain an error
+message.
+
+Lastly, the set of available network providers is listed under `providers`.
+Each of them is referenced by the Public land mobile network (PLMN) code, identifying a country,
+and a mobile network operator in that country. If the modem is connected to one of the networks,
+that network will have attribute `current-serving` returned as `true`.
+
+### WWAN Metrics API endpoint
+
+JSON-formatted WWAN-related metrics can be requested by applications on the
+`/eve/v1/wwan/metrics.json` endpoint.
+
+For example:
+
+```shell
+curl 169.254.169.254/eve/v1/wwan/metrics.json 2>/dev/null | jq
+{
+  "networks": [
+    {
+      "logical-label": "cell-modem0",
+      "physical-addrs": {
+        "interface": "wwan0",
+        "usb": "1:1",
+        "pci": "0000:00:1d.7"
+      },
+      "packet-stats": {
+        "rx-bytes": 504,
+        "rx-packets": 6,
+        "rx-drops": 0,
+        "tx-bytes": 504,
+        "tx-packets": 6,
+        "tx-drops": 0
+      },
+      "signal-info": {
+        "rssi": -60,
+        "rsrq": -13,
+        "rsrp": -87,
+        "snr": 116
+      }
+    }
+  ]
+}
+```
+
+The underlying structure, used by EVE to store the information and output it as JSON,
+is named `WwanMetrics` and can be found in [zedroutertypes.go](../pkg/pillar/types/zedroutertypes.go).
+
+The endpoint returns a list of entries, one for every cellular modem, with the modem's
+logical label (from the device model) used as a reference. Just like in the
+[WWAN status API endpoint](#wwan-status-api-endpoint), the physical connection between
+the device and the cellular modem is described by the `physical-addrs` structure.
+
+Packet statics contain RX/TX packet/byte counters (all `uint64`) as recorded by the modem itself.
+This may differ from the Linux kernel counters (from `networkMetric` proto message) if, for example,
+some packets were dropped by the modem.
+
+Cellular signal strength is described using multiple different measurements:
+
+- Received signal strength indicator (RSSI) measured in dBm (decibel-milliwatts)
+- Reference Signal Received Quality (RSRQ) measured in dB (decibels)
+- Reference Signal Receive Power (RSRP) measured in dBm (decibel-milliwatts)
+- Signal-to-Noise Ratio (SNR) measured in dB (decibels)
```

Comment:
Shouldn't this say dB*10 i.e., 11.6 dB is represented as 116 based on what you said?

Reply:
No because as I wrote above, with the intention to not confuse users I decided that it is probably better to divide by ten and round it up. So it is published as integer in dB.

Reply:
OK